meta/gensairpc.README

```diff
@@ -0,0 +1,34 @@
+NAME
+    gensairpc.pl - generate RPC interface of SAI for PTF
+
+DESCRIPTION
+    This script generates RPC interface of SAI function for PTF tests. It
+    should be used after SAI interface update.
+
+    For more details see the development documentation (rpc/README.md).
+
+USAGE
+    In order to run, just call:
+
+      ./gensairpc.pl
+
+    Before committing it is a good idea to clean before generation:
+
+      ./gensairpc.pl --clean-meta
+
+    In order to generate debug files and avoid cleanup, use --dbg or --dump
+    flags. Use --experimental to generate extensions and experimental code.
+
+    For more information, use --help:
+
+      ./gensairpc.pl --help
+
+DEPENDENCIES
+    Ubuntu packages:
```

Comment:
is there any place that mentioned 'graphviz' as a dependency for 'dot' which required in Makefile 'BINS = doxygen perl dot'
Not sure, if libmoose-perl needed

Reply:
'BINS = doxygen perl dot' is an old part of Makefile. We don't introduce 'dot' with the generator. That's why its dependencies are not listed here.

Reply:
\+ IMO we could add libmoose-perl to the list, but it is a dependency of libmoosex-aliases-perl so they should be installed together.